Taskfile.yml

```diff
@@ -1024,6 +1024,13 @@ tasks:
     cmds:
       - "{{.SCRIPTS_ROOT}}/check_frontmatter.py"
 
+  doc:dependencies:
+    desc: Generate documentation for depencencies
+    cmds:
+      - cp ./docs/v2/dependencies-header.md ./docs/hugo/content/contributing/dependencies.md
```

Comment:
Does `cp` overwrite the file with no prompting if it already exists? I guess yes unless you use `-i`?

Reply:
Yes.